components/infra-proxy-service/migrations/pipeline/utility.go

```diff
@@ -72,6 +73,18 @@ func createProjectFromOrgIdAndServerID(ctx context.Context, serverId string, org
 		SkipPolicies: false,
 	}
 
+	existingProject := &authz.GetProjectReq{
+		Id: serverId + "_" + orgId,
+	}
+
+	projResp, err := authzProjectClient.GetProject(ctx, existingProject)
+	if err != nil {
+		log.Error("cannot get the project with the id: ", existingProject.Id)
+	}
+	if projResp != nil {
```

Comment:
If the project already exists we should not return error but skip Create project